### DISCUSSION TITLE
domino/14.0.0/admin/secu_exportingcertstofile

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#151] [Q&A]
# domino/14.0.0/admin/secu_exportingcertstofile

You can export the credentials in a TLS Credentials document to a file.

https://help.hcl-software.com/domino/14.0.0/admin/secu_exportingcertstofile.html

**Comment by chaitu4c0** [comment]
Hello team,

In step 4, it is mentioned to specify the path for the exported file.
However, if you specify the folder path only (ex: C:\certs), then it reports error messages.
It is necessary to specify the certificate file name with extension (.p12,.PEM,etc..) to export TLS credentials.
Please update the step 4 accordingly in the documentation.
